GiraffeApp/Antiforgery.fs

```diff
@@ -0,0 +1,30 @@
+module Antiforgery
+
+open Microsoft.AspNetCore.Http
+open Microsoft.AspNetCore.Antiforgery
+open Giraffe
+open Giraffe.ViewEngine
+
+// Generates a CSRF token using the Microsoft.AspNetCore.Antiforgery package,
```

Comment:
Teerawat, I copied this code from https://github.com/pimbrouwers/Giraffe.Antiforgery.  The main code doesn't call it yet.  It doesn't seem to fit.